### PR TITLE
Move location options to a codelist

### PR DESCRIPTION
**Pull request by lagartoverde**
A few things to take into account, 
this codelist is done on dev, but it will need to be done in other environments and set the uri in the settings of the plugin.
As it's a codelist it will still appear for the users to select in mow when they select codelist (this can be filtered out in the frontend but I don't think is really necessary.
I also made the modifications on the dev codelist.

### CODE REVIEW

**Comment by lagartoverde** [comment]
Ahh I also made the variables from the location and any variable inside a codelist option appear with a mark highlight manual to make it easier to replace

**Review by benjay10 — APPROVED** [review]
Me being out of touch with what this PR is trying to solve, I don't know how to properly review this. Everything works, not much different from how it was before, so that is probably a good thing.

File: addon/commands/insertAndCollapse.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    return true;
+  }
+
+  execute(controller, htmlString, node) {
+    const range = controller.rangeFactory.fromInNode(
+      node,
+      0,
+      node.getMaxOffset()
+    );
+    controller.executeCommand('insert-html', htmlString, range);
```
Comment (nvdk) [inline addon/commands/insertAndCollapse.js:18]: were commands supposed to call other commands? do we have other instances where we do this?

**Comment by nvdk** [comment]
I'd expect a PR with a migration on https://github.com/lblod/app-mow-registry to add the data to the other environments?

File: addon/commands/insertAndCollapse.js (hunk 1 above)
Comment (lagartoverde) [reply addon/commands/insertAndCollapse.js:18]: This was a discussion with Arne at some point, we decided that they could I think, but we can still discuss it

**Comment by lagartoverde** [comment]
https://github.com/lblod/app-mow-registry/pull/52 Created the PR now as yesterday I couldn't due to Github issues and I forgot until now

**Review by nvdk — CHANGES_REQUESTED** [review]
This doesn't seem to work for me, I see 
```
query: PREFIX lblodMobilitiet: <http://data.lblod.info/vocabularies/mobiliteit/>
    PREFIX dct: <http://purl.org/dc/terms/>
    PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
    SELECT DISTINCT * WHERE { 
      <undefined> a lblodMobilitiet:Codelist.
      ?codelistOptions skos:inScheme <undefined>.
      ?codelistOptions skos:prefLabel ?label.
      OPTIONAL {
        <undefined> dct:type ?type.
      }
    }
 ```
 as a query

**Review by nvdk — APPROVED** [review]
looks like I need to restart the server to fetch environment.js changes, does work as expected

**Pull request merged**